app/views/shared/_sidenav.html.erb

```diff
@@ -43,7 +43,7 @@
         </ul>
       <%- end -%>
     </li>
-    <li><%= link_to 'Entitlements', page_path('entitlements'), class: current_page?(id: 'entitlements') ? 'usa-current' : nil %></li>
+    <li><%= link_to 'Entitlements', page_path('entitlements'), class: controller_name == 'entitlements' ? 'usa-current' : nil %></li>
```

Comment:
@tpitale As best as I can tell, `current_page?` won't properly answer the question, "Does this page fall under controller `foo`?" A couple of examples:

```
<%= current_page?(controller: 'entitlements') %>
```

…will return `true` at `localhost:3000/entitlements`, but will return `false` at `localhost:3000/entitlements/e-1`. It'll also blow up on _other_ pages:

```
No route matches {:action=>"index", :controller=>"high_voltage/entitlements", :id=>"moving-guide"}
```

☝️ That _might_ be revealing issues elsewhere in this file, though, that I should look at refactoring.

🤔